csvkit/utilities/csvjson.py

```diff
@@ -110,39 +144,48 @@ def dump_json(data, newline=False):
                 feature['type'] = 'Feature'
                 properties = OrderedDict()
                 geoid = None
+                geo_type = 'Point'
                 lat = None
                 lon = None
+                coords = None
 
                 for i, c in enumerate(row):
                     if i == lat_column:
+                        if not c:
```

Comment:
Ah, nevermind, I confused this with a different comment you made about lat/lon being empty.